examples/kotlin-diktat/run.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+#wget -nc https://github.com/pinterest/ktlint/releases/download/0.39.0/ktlint
+wget -nc https://repo1.maven.org/maven2/com/pinterest/ktlint/0.39.0/ktlint-0.39.0.jar
+wget -nc https://github.com/cqfn/diKTat/releases/download/v0.6.2/diktat-0.6.2.jar
+# Download latest release of SAVE CLI
+#wget -nc https://github.com/cqfn/save/releases/download/v0.1.0-alpha.2/save-0.1.0-alpha.2-linuxX64.kexe -O save
+# use this for mingw
```

Comment:
Probably we can use `"$OSTYPE"`
https://stackoverflow.com/a/8597411

Reply:
Thx, added